### DISCUSSION TITLE
Can we have a way to disable ALL of the HTTPS stuff?

### DISCUSSION THREAD

**Discussion by VictorioBerra** [Dotnet-Boxed/Templates#210] [Q&A]
We host behind a load balancer which does our SSL termination. From there, its just IIS hosting + kestrel.

So we do not need all the HSTS, Cert, Headers stuff. Is there a way to fully pull all of that out?

**Comment by RehanSaeed** [comment]
Just pass `--https-everywhere false` as an argument and that will remove all HTTPS code.
